Makefile

```diff
@@ -17,6 +17,7 @@ NAME = "slo_generator"
 
 PIP=pip3
 PYTHON=python3
+MYPY=mypy
```

Comment:
`mypy` is only called once so we don't need this variable. Just call it directly in the target.

Reply:
Done